easybuild/easyblocks/a/abaqus.py

```diff
@@ -35,14 +35,14 @@
 import glob
 import os
 from collections import OrderedDict
-from distutils.version import LooseVersion
 
 from easybuild.easyblocks.generic.binary import Binary
 from easybuild.framework.easyblock import EasyBlock
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.tools.build_log import EasyBuildError
 from easybuild.tools.environment import setvar
 from easybuild.tools.filetools import change_dir, symlink, write_file
+from easybuild.tools.loose_version import LooseVersion
```

Comment:
Please use `from easybuild.tools import LooseVersion` instead, like we do everywhere in framework?

Reply:
(I think this was just a test due to the import not working in the CI)